src/main/cpp/Robot.cpp

```diff
@@ -25,6 +25,11 @@ void Robot::RobotInit() {
   HatchIntake::getInstance();
 }
 
+constexpr void Robot::LimitSwitches() {
+  frc::SmartDashboard::PutBoolean("Arm Reverse Limit Switch", HatchIntake::getInstance()->IsHatchBack());
+	frc::SmartDashboard::PutBoolean("Arm Forward Limit Switch", HatchIntake::getInstance()->IsHatchForward());
```

Comment:
I think these labels should more closely match what they are indicating. For example "Hatch Is Forward", "Hatch is Reverse".